Documentation/Doxygen/doxygen.bib

```diff
@@ -1550,15 +1550,6 @@ @article{zijdenbos1994
   url          = {https://doi.org/10.1109/42.363096}
 }
 
-@inproceedings{Aylward_2016,
```

Comment:
Should this reference be somehow added to Ultrasound remote module?

Reply:
This publication is already listed in the US remote modules [README.rst references section](https://github.com/KitwareMedical/ITKUltrasound/blob/master/README.rst#references).